stdlib/http/src/main/ballerina/http/non_listening_service_endpoint.bal

```diff
@@ -43,7 +43,7 @@ public type MockListener object {
 
     public function init (ServiceEndpointConfiguration c);
     public extern function initEndpoint () returns (error?);
-    public extern function register (service s, map<any> annotationData) returns error?;
+    public extern function register (service s, string? name) returns error?;
```

Comment:
@wggihan why is this a public function? Shall we make this module private.

Reply:
@chamil321 Can you please check the feasibility of making this private?

Reply:
@wggihan Sure I'll check